### PR TITLE
Metrics test

### PR DESCRIPTION
**Pull request by rodriquelca**
## Issue & Reproduction Steps

Expected behavior: 
write a test file to get metrics in order to evaluate performance
Actual behavior: 

## Solution
- cypress tes was done
## How to Test
run the test /home/rodrigo/dev2/screen-builder/tests/e2e/specs/Metrics2.spec.js

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7235

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

[FOUR-7235]: https://processmaker.atlassian.net/browse/FOUR-7235?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/e2e/specs/Metrics2.spec.js [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+        logText =
+          "Characters Number: " +
+          text.length +
+          " =>  Time: " +
+          measure +
+          "ms \n";
+        cy.log(logText).then(() => {
+          return element;
+        });
+        cy.writeFile("tests/e2e/fixtures/results.txt", logText, {
```
Comment (caleeli) [inline tests/e2e/specs/Metrics2.spec.js:385]: `tests/e2e/fixtures` folder is not stored by circleci

You could try to add an additional store_path in `.circleci/config.yml` configuration file, like:
```
    - store_artifacts:
        path: tests/e2e/screenshots
    - store_artifacts:
        path: tests/e2e/metrics
```
see also: https://circleci.com/docs/configuration-reference/#storeartifacts

Or you could try store it to `tests/e2e/screenshots`

Comment (caleeli) [reply tests/e2e/specs/Metrics2.spec.js]: If you want to store to `tests/e2e/metrics`
you must create the folder before run test, (in `.circleci/config.yml`):

```
    - run: mkdir tests/e2e/metrics
    - run:
        name: run tests
```

**Review by caleeli — APPROVED** [review]
Great test!

**Comment by pmkerberos** [comment]
## Typing speed test
(Delay: 50, Characters Number: 10) =>  Total Time: 686ms 
(Delay: 50, Characters Number: 20) =>  Total Time: 1132ms 
(Delay: 50, Characters Number: 30) =>  Total Time: 1681ms 
(Delay: 50, Characters Number: 40) =>  Total Time: 2213ms 
(Delay: 50, Characters Number: 50) =>  Total Time: 2758ms 
Time per Character: 52.25ms 
Screen-builder delay per Character: 2.25ms

**Comment by pmkerberos** [comment]
## lighthouse test
**Loop6 multi-column select lists rich texts text areas**

performance record is 15 and threshold was 12
accessibility record is 95 and threshold was 50
interactive record is 11824.252549999994 and threshold was 16000

**Comment by pmkerberos** [comment]
## lighthouse test
**Loop6 multi-column select lists rich texts text areas validations rules visibility rules and submit button**

performance record is 15 and threshold was 12
accessibility record is 95 and threshold was 50
interactive record is 11313.480149999996 and threshold was 16000

**Pull request merged**